### PR TITLE
Universal SQL: Misc Fixes

### PR DESCRIPTION
**Pull request by ItsMeBrianD**
### Description

While writing the SQL Console component, I encountered a few issues that were easily resolved

### Checklist

- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)

### CODE REVIEW

File: packages/preprocess/src/process-queries.cjs [hunk 1]
```diff
@@ -247,6 +227,27 @@ const createDefaultProps = function (filename, componentDevelopmentMode, duckdbQ
             }
         });
 
+		import { browser, dev } from "$app/environment";
```
Comment (csjh) [inline packages/preprocess/src/process-queries.cjs:230]: Why was this moved outside `queryDeclarations`?

**Review by csjh — APPROVED** [review]

File: packages/preprocess/src/process-queries.cjs (hunk 1 above)
Comment (ItsMeBrianD) [reply packages/preprocess/src/process-queries.cjs:230]: Because we want it to always be available; I had to put a dummy query on my page with the sql console so that the query deps would actually load onto the page
